### PR TITLE
Fixed helm test sample.

### PR DESCRIPTION
**Pull request by WilliamDenniss**
The current sample doesn't work as documented.

New sample based on https://github.com/kubernetes/charts/pull/1156.

### CODE REVIEW

**Review by thomastaylor312 — APPROVED** [review]
This is good to merge once https://github.com/kubernetes/charts/pull/1156 is merged. Thank you for improving the example

File: docs/chart_tests.md [hunk 1]
```diff
@@ -34,19 +34,33 @@ mariadb/
   templates/
   templates/mariadb-tests.yaml
```
Comment (michelleN) [inline docs/chart_tests.md]: We'll probably want to change this chart structure also to match the example below.

**Review by michelleN — CHANGES_REQUESTED** [review]

**Comment by michelleN** [comment]
oh awesome. I like this example way better. The original example was never meant to work on the actual stable/wordpress chart. It was just meant to show what a test might actually look like and where it might live in a chart. Seeing an actual example makes lots of sense though. We should add a link to the stable/wordpress chart as well in case the example changes in the kubernetes/charts repo. People will be able to see the updated tests if it ends up changing.

**Comment by technosophos** [comment]
Can someone provide an update on this one?

**Comment by michelleN** [comment]
@technosophos I had requested some changes but they were minor. Happy to approve this one and make the one change I had in mind in a following PR.

**Review by michelleN — APPROVED** [review]

**Comment by thomastaylor312** [comment]
@technosophos I was waiting for the example in https://github.com/kubernetes/charts/pull/1156 to be merged so they matched. If you are ok with not waiting, feel free to merge this

**Comment by WilliamDenniss** [comment]
Sorry for the delay. Give me a day and I'll address the comment.

File: docs/chart_tests.md (hunk 1 above)
Comment (WilliamDenniss) [inline docs/chart_tests.md]: Fixed. Also removed the `LICENSE` reference as that file isn't the chart's folder.

**Comment by WilliamDenniss** [comment]
Thanks for your reviews!

@michelleN glad you like my new sample. I addressed your comments.

@thomastaylor312 I'd suggest merging now. The WordPress chart never had any tests in the repo, so updating the docs first won't result in them being out of sync, and the existing doc sample is non-functional, so I think this change is a definitive improvement.

To test kubernetes/charts#1156 yourself:
```
git clone https://github.com/kubernetes/charts.git
cd charts
git fetch origin pull/1156/head:pull1156
git checkout pull1156

helm init
cd charts/stable/wordpress
helm install . --name=wp
helm status wp
# wait for all Deployments to be fully Available
helm test wp
helm delete wp
```

**Comment by thomastaylor312** [comment]
I'll go ahead and merge this

**Pull request merged**